src/vm/winrttypenameconverter.cpp

```diff
@@ -828,7 +828,7 @@ bool WinRTTypeNameConverter::IsRedirectedWinRTSourceType(MethodTable *pMT)
 // Get TypeHandle from a WinRT type name
 // Parse the WinRT type name in the form of WinRTType=TypeName[<WinRTType[, WinRTType, ...]>]
 //
-TypeHandle WinRTTypeNameConverter::GetManagedTypeFromWinRTTypeName(LPCWSTR wszWinRTTypeName, bool *pbIsPrimitive)
+TypeHandle WinRTTypeNameConverter::LoadManagedTypeForWinRTTypeName(LPCWSTR wszWinRTTypeName, bool *pbIsPrimitive, ICLRPrivBinder* loadBinder /* = nullptr */)
```

Comment:
Please fix the order of parameters for this function signature with respect to ins and outs.